File: app/src/main/java/me/wpkg/cli/android/ClientSelectedListener.java
package me.wpkg.cli.android;

import androidx.constraintlayout.widget.ConstraintLayout;
import me.wpkg.cli.json.JsonMaps;

public interface ClientSelectedListener
{
    void clientSelected(JsonMaps.ClientObject client);

    void clientMenuInvoked(JsonMaps.ClientObject client, ConstraintLayout layout);
}

File: app/src/main/java/me/wpkg/cli/android/ClientSelectedListener.kt
package me.wpkg.cli.android

import androidx.constraintlayout.widget.ConstraintLayout

import me.wpkg.cli.json.JsonMaps.ClientObject

interface ClientSelectedListener
{
    fun clientSelected(client: ClientObject)
    fun clientMenuInvoked(client: ClientObject, layout: ConstraintLayout)
}